controllers/namespace/namespace_suite_test.go

```diff
@@ -0,0 +1,82 @@
+/*
+Copyright 2022.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package namespace_test
```

Comment:
The content of the namespace_suite_test.go has changed and is much more now. I think this is created automatically in some way. Am thinking if we can revert to the content of it to the old version as such?

Reply:
revert the namespace_suite_test.go to the old version